RELEASE/scripts/autoscend/paths/path_of_the_plumber.ash

```diff
@@ -425,3 +425,42 @@ boolean plumber_equipTool(stat st)
 	}
 	return false;
 }
+
+void plumber_eat_xp()
+{
+	//eat stuff for XP.
+	if(!in_plumber() || fullness_left() < 1)
+	{
+		return;
+	}
+	if(!prepare_food_xp_multi())
+	{
+		return;		//we are not prepared.
+	}
+	
+	//TODO diabolic pizza oven with pie man was not meant to eat
+	
+	item milk = $item[gallon of milk];
+	if(creatable_amount(milk) > 0 || item_amount(milk) > 0 || canPull(milk))
```

Comment:
Should check for 15 fullness available as well. If/when other foods are added, it isn't a sure thing we will have an empty stomach

Reply:
Good point.